src/features/favorites/atoms/Buttons/Sort.tsx

```diff
@@ -5,17 +5,17 @@ import LinearGradient from 'react-native-linear-gradient'
 import styled from 'styled-components/native'
 
 import { UseNavigationType } from 'features/navigation/RootNavigator'
-import { TouchableOpacity } from 'ui/components/TouchableOpacity'
 import { Sort as SortIconDefault } from 'ui/svg/icons/Sort'
 import { getSpacing, Spacer, Typo } from 'ui/theme'
+import { TouchableLink } from 'ui/web/link/TouchableLink'
 
 export const Sort: React.FC = () => {
   const { navigate } = useNavigation<UseNavigationType>()
   function onPress() {
     navigate('FavoritesSorts')
   }
   return (
-    <Container onPress={onPress} testID="SortButton">
+    <Container to={{ screen: 'FavoritesSorts' }} onPress={onPress} testID="SortButton">
```

Comment:
J'ai l'impression que ça fais duplicate avec le onPress, pourquoi ?

Reply:
Oui c'est le cas mais si on veut render un anchor on est obligé de passer la props `to` pour générer le `href`

Reply:
Est-ce que tu peux avoir la liste complète des composants utiliser de la même façon ? Une abstraction reste possible.

Reply:
Je comprends que ce n'est pas simple surtout si les `onPress` ne font pas que navigate, mais on peut toujours avoir un `onPress` qui précède le navigate et qui déclenche le navigate si pas d'erreur. Bien sur cela couterai un composant avec pas mal de props, mais je me demande si sur le très long terme, cette abstraction ne serait pas plus bénéfique. Pour l'instant je ne suggère pas de changer, juste d'évaluer.

Reply:
`TouchableLink` et `AppButton` (et les composants qui les utilisent, tu veux cette liste aussi ?)